Doc/library/functions.rst

```diff
@@ -1562,7 +1562,7 @@ are always available.  They are listed here in alphabetical order.
    about strings, see :ref:`textseq`.
 
 
-.. function:: sum(iterable[, start])
+.. function:: sum(iterable, /[, start])
```

Comment:
Maybe `start=0` ?

Reply:
Done in e035eee

I have eliminated the text "*start* defaults to ``0``" as is now redundant.